README.md

```diff
@@ -109,7 +158,8 @@ jobs:
           access-control-list-json: >
             [
               {
-                "users": "Can View"
```

Comment:
The example we had for `access-control-list-json` did not follow expected input format for: https://docs.databricks.com/dev-tools/api/latest/jobs.html#operation/JobsRunsSubmit